client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -175,6 +176,8 @@ const LowerJaw = ({
       );
   };
 
+  const showDesktopButton = window.innerWidth > MAX_MOBILE_WIDTH;
```

Comment:
Wouldn't `window. navigator.userAgent` be better to determine type of device instead of using viewport size?

Reply:
I agree, we have been traditionally altering the layout for smaller screens regardless of the device to save up space. 
That being said, we are missing out on some optimization opportunities. 

This pr could go in as is and we could discuss this in a separate issue and research best practices.